src/Database/Driver.php

```diff
@@ -994,6 +1002,26 @@ public function getRole(): string
         return $this->_config['_role'] ?? Connection::ROLE_WRITE;
     }
 
+    /**
+     * Enable query logging.
+     *
+     * @return void
+     */
+    public function enableQueryLogging(): void
```

Comment:
`enableAutoQuoting()/disableAutoQuoting()` return `$this`, so it would be better to do the same for these 2 methods for API consistency.